cluster/common.sh

```diff
@@ -1002,7 +1002,6 @@ function create-certs {
   PRIMARY_CN="${primary_cn}" SANS="${sans}" generate-certs
   AGGREGATOR_PRIMARY_CN="${primary_cn}" AGGREGATOR_SANS="${sans}" generate-aggregator-certs
 
-  CERT_DIR="${KUBE_TEMP}/easy-rsa-master/easyrsa3"
```

Comment:
shouldn't CERT_DIR and AGGREGATOR_CERT_DIR be moved from assumed vars to vars set? Why do you need to move these?

Reply:
I'm not sure what you mean,  they were declared as set by `create-certs` and that does not change - they still are set by `create-certs`, but indirectly (by a function it calls: `setup-easyrsa`).
I needed to move setting of these variable to before `generate-certs` and `generate-aggregator-certs` are called. I could have done this either by moving these lines to before `setup-easyrsa` invocation, or *into* its body. I chose the latter since it moves the setting closer to the place where these directories are actually created, to decrease the overall cognitive burden.

I hope that answers your question.

Reply:
I mean that the function comment is now wrong. It says that CERT_DIR is set by this function when it's not.

Reply:
I don't understand what the semantics of that comment should be, then. To a caller it appears that this function sets the variables. Does it matter that it's actually set deeper in the call stack?

Reply:
Ok, I see what you mean.